boot/raspi3/BUILD.go

```diff
@@ -0,0 +1,46 @@
+package raspi3
+
+import (
+	"dbt-rules/RULES/core"
+	"dbt-rules/RULES/util"
+
+	gcc "aarch64-elf-gcc"
+	"libsupcxx/RULES/boot"
+	"libsupcxx/RULES/config"
+	"libsupcxx/RULES/libsupcxx/lib"
+)
+
+var crt2 = util.CopyFile{
+	From: gcc.CrtBegin,
+	To:   out("crt2.o"),
+}
+
+var crt3 = util.CopyFile{
+	From: gcc.CrtEnd,
+	To:   out("crt3.o"),
+}
+
+var BootFirst = lib.Library{
```

Comment:
can `BootFirst` and `BootLast` be private?

Reply:
Yeah they can be

Reply:
On second thought, the reason I left them public is that I wasn't sure whether we'd want to change the way private targets are handled in dbt. They can be private for now tho.